### ISSUE TITLE
Images are not shown in published build

### ISSUE THREAD

**Issue by shinichy** [tiensonqin/exponent-cljs-template#51]
# Issue
Once you build with lein prod-build and publish to an expo url, the app once downloaded does not display any images at all.
I think the same issue as https://github.com/seantempesta/expo-cljs-template/issues/23 is happening again.

# Steps to reproduce:
`lein new expo sample && cd sample && yarn install && lein prod-build && exp publish`

**Comment by seantempesta** [comment]
Confirmed.  I'll have to dig into this to figure out what's going on.  Some observations:

During publish, it doesn't appear any assets are uploaded
```
[exp] Uploading assets
[exp] No assets changed, skipped.
[exp] Uploading JavaScript bundles
[exp] Published
```
Searching the source reveals the asset did survive `prod` compilation and should have been detected by expo's `require` scan:
```
sean@sean-mac ~/s/sample> grep -ri "cljs.png" *
main.js:function kp(){return function(a){return function(){return new X(null,5,5,Y,[gp,new r(null,1,[Xi,new r(null,3,[pk,"column",xk,40,Bh,"center"],null)],null),new X(null,2,5,Y,[hp,new r(null,2,[Li,**require("./assets/images/cljs.png"**),Xi,new r(null,2,[yi,200,qk,200],null)],null)],null),new X(null,3,5,Y,[fp,new r(null,1,[Xi,new r(null,4,[Vh,30,bi,"100",ck,20,Nh,"center"],null)],null),D(a)],null),new X(null,3,5,Y,[ip,new r(null,2,[Xi,new r(null,3,[ri,"#999",rj,10,Wj,5],null),fh,function(){return function(){return jp.alert("HELLO!")}}(a)],
```

**Comment by nikki93** [comment]
Does this happen on the default template?

**Comment by pyankoff** [comment]
Having the same issue on newly created project.

**Comment by seantempesta** [comment]
@nikki93 It appears so.  I've been working on an intense project the past few months and haven't had enough time to dig into this.  I'm going to try upgrading to the latest expo and I'll post if it fixes this behavior.

**Comment by seantempesta** [comment]
@nikki93:  Upgrading to the latest expo version (25) doesn't work, neither does explicitly adding offline asset inclusion:
```js
    "assetBundlePatterns": [
      "assets/images/**",
      "assets/icons/**"
    ]
```
Maybe there is an expo regex that isn't matching the output created from the Clojurescript compiler?  Do you know where I could look to debug this further?  The regex needs to be able to match this:
```
[cljs.core.cst$kw$source,require("./assets/images/cljs.png")
```

**Comment by seantempesta** [comment]
Adding diagnostics report:

sean@sean-mac ~/s/expo-reagent-test> exp diagnostics
[exp] Generating diagnostics report...
[exp] You can join our slack here: https://slack.expo.io/.

Environment:
  OS: macOS High Sierra 10.13.3
  Node: 9.7.1
  Yarn: 1.5.1
  npm: 5.6.0
  Watchman: 4.9.0
  Xcode: Xcode 9.3 Build version 9Q98q
  Android Studio: Not Found

Packages: (wanted => installed)
  expo: ^25.0.0 => 25.0.0
  react: 16.2.0 => 16.2.0
  react-native: https://github.com/expo/react-native/archive/sdk-25.0.0.tar.gz => 0.52.0

Diagnostics report:
  https://exp-xde-diagnostics.s3.amazonaws.com/seantempesta-8dc1ad08-727c-4b86-9356-0e9b34093d4a.tar.gz

**Comment by tkamat** [comment]
OK this seems to be fixed for me by #59

**Comment by seantempesta** [comment]
Just integrated #59, so this should be fixed.  Please reopen if it's not working.
